apps/system-e2e/src/support/utils.ts

```diff
@@ -1,30 +1,20 @@
-//import PDFDocument from 'pdfkit'
 import { debuglog } from 'util'
 import fs from 'fs'
+import { getLogger } from '../../../../infra/src/logging'
 
 export const sleep = (ms: number) => new Promise((r) => setTimeout(r, ms))
+export const logger = getLogger()
+export const debug = logger.debug
 
-export function createMockPdf() {
+export const createMockPdf = () => {
   fs.writeFile('./mockPdf.pdf', 'test', (e) => {
```

Comment:
This creates a non-async function that calls a function with a hook and returns before file is created. Can this create an error?

Reply:
Probably. It's being used in tests, and they aren't failing :shrug: 

This should be revisited when lib-ified